packages/react/src/patterns/blocks/CardArray/CardArray.js

```diff
@@ -0,0 +1,114 @@
+/**
```

Comment:
@annawen1 was this here before? I think @RaphaelAmadeu already refactored this...

Reply:
uhhhh not sure, there was a bunch of updating from master that I pulled in a few times

Reply:
ok removed